readme.md

```diff
@@ -121,12 +122,14 @@ To learn how to use the new features in this release, please see the wiki pages
 - Fix handling of amp-bind attributes to ensure that `“>”` can appear inside attribute values. See [#1119](https://github.com/Automattic/amp-wp/pull/1119). Props westonruter.
 - Tree-shake CSS selectors for HTML elements that target non-active languages. See [#1221](https://github.com/Automattic/amp-wp/pull/1221). Props westonruter.
 - Redirect to post list table in case of admin bar validate request failure. See [#1229](https://github.com/Automattic/amp-wp/pull/1229). Props westonruter.
+- Fix expected output for `test_decorate_shortcode_and_filter_source`. See [#1535](https://github.com/Automattic/amp-wp/pull/1535). Props westonruter.
```

Comment:
I'm going to replace this item with a line item for #1584. This unit test fix I don't think necessarily needs its own changelog entry.

Reply:
Sounds good.